sql/plan/ddl.go

```diff
@@ -99,10 +98,28 @@ func (c *CreateTable) WithDatabase(db sql.Database) (sql.Node, error) {
 	return &nc, nil
 }
 
+// Schema implements the sql.SchemeModifiable interface.
```

Comment:
Please remove all these leftover Schema() methods and comments

Reply:
Just the comments need to be updated, as the `Schema()` functions themselves need to stay (and are a part of `sql.Node`). I already removed the `WithSchema()` methods.